src/bank.rs

```diff
@@ -296,19 +302,21 @@ impl Bank {
     }
 
     /// Process an ordered list of entries.
-    pub fn process_entries<I>(&self, entries: I) -> Result<()>
+    pub fn process_entries<I>(&self, entries: I) -> Result<usize>
     where
         I: IntoIterator<Item = Entry>,
     {
         for entry in entries {
+            self.entry_count.fetch_add(1, Ordering::Relaxed);
+
             if !entry.transactions.is_empty() {
                 for result in self.process_transactions(entry.transactions) {
                     result?;
                 }
             }
             self.register_entry_id(&entry.id);
         }
-        Ok(())
+        Ok(self.entry_count())
```

Comment:
@rob-solana, It's strange to both return this value and set a state variable. You're offering two ways to get the same information. I'd recommend getting rid of the state variable.

Reply:
the return value is a nice info thingie.  the state variable is the real useful one: it's pulled out later for server/streamer/window construction